v2/internal/attacktechniques/aws/defense-evasion/dns-delete/main.tf

```diff
@@ -0,0 +1,49 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0.0"
+    }
+  }
+}
+provider "aws" {
+  skip_region_validation      = true
+  skip_credentials_validation = true
+  default_tags {
+    tags = {
+      StratusRedTeam = true
+    }
+  }
+}
+
+resource "random_string" "suffix" {
+  length    = 10
+  min_lower = 10
+  special   = false
+}
+
+locals {
+  resource_prefix = "stratus-red-team-dns-delete"
+}
+
+locals {
+  bucket-name = "${local.resource_prefix}-bucket-${random_string.suffix.result}"
+}
+
+resource "aws_route53_resolver_query_log_config" "config" {
+  name            = "${local.resource_prefix}-config-${random_string.suffix.result}"
+  destination_arn = aws_s3_bucket.query_log.arn
+}
+
+resource "aws_s3_bucket" "query_log" {
```

Comment:
I don't believe we do. This doesn't actually end up sending logs since it doesn't associate any VPC's with the configuration, which is where the logs are sourced from. So it's OK to not have IAM, unless we want to also test associating & dissociating VPC's from the logging configuration.

Reply:
Was mainly wondering if `terraform apply` would succeed here. If so, this LGTM!

Reply:
Yup `terraform apply` works :)